src/js/lib/jquery.kazoosdk.js

```diff
@@ -525,6 +525,12 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			crm: {
+				'getIntegration': { verb: 'GET', url: 'accounts/{accountId}/crm_integration'},
+				'setIntegration': { verb: 'PUT', url: 'accounts/{accountId}/crm_integration'},
```

Comment:
Is the intent here to create an integration or does set has a different meaning? If it is, we should use `create` to keep that action uniform throughout the sdk.
```suggestion
				'createIntegration': { verb: 'PUT', url: 'accounts/{accountId}/crm_integration'},
```

Reply:
The intent is to create a new integration, I'll fix the name.